WorkloadManagementSystem/Service/WMSUtilities.py

```diff
@@ -77,29 +70,31 @@ def getCREAMPilotOutput( proxy, pilotRef, pilotStamp ):
   result['StdErr'] = error
   return result
 
-def theARCJob(theCE, theArcID):
-  # Create an ARC Job with all the needed / possible parameters defined.
-  # By the time we come here, the environment variable X509_USER_PROXY should already be set
+def ARCJob( theCE, theArcID ):
+  """ Create an ARC Job with all the needed / possible parameters defined.
+      By the time we come here, the environment variable X509_USER_PROXY should already be set
+  """
   j = arc.Job()
   j.JobID = theArcID
-  statURL = "ldap://%s:2135/Mds-Vo-Name=local,o=grid??sub?(nordugrid-job-globalid=%s)" % (theCE, theArcID)
-  j.JobStatusURL = arc.URL(statURL)
+  statURL = "ldap://%s:2135/Mds-Vo-Name=local,o=grid??sub?(nordugrid-job-globalid=%s)" % ( theCE, theArcID )
```

Comment:
I leave @rajanandakumar to take care of these types of issues, since with this PR I am just fixing some obvious bugs